ui/src/main/java/edu/wpi/grip/ui/pipeline/AddSourceView.java

```diff
@@ -164,7 +164,7 @@ private void loadCamera(Dialog<ButtonType> dialog, SupplierWithIO<CameraSource>
             } catch (IOException e) {
                 // This will run it again with the new values retrieved by the supplier
                 failureCallback.accept(e);
-                Platform.runLater(() -> loadCamera(dialog, cameraSourceSupplier, failureCallback));
+                loadCamera(dialog, cameraSourceSupplier, failureCallback);
```

Comment:
Yes

On Mon, Dec 14, 2015, 6:33 PM Jonathan Leitschuh <notifications@github.com>
wrote:

> In ui/src/main/java/edu/wpi/grip/ui/pipeline/AddSourceView.java
> <https://github.com/WPIRoboticsProjects/GRIP/pull/213#discussion_r47576399>
> :
>
> > @@ -164,7 +164,7 @@ private void loadCamera(Dialog<ButtonType> dialog, SupplierWithIO<CameraSource>
> >              } catch (IOException e) {
> >                  // This will run it again with the new values retrieved by the supplier
> >                  failureCallback.accept(e);
> > -                Platform.runLater(() -> loadCamera(dialog, cameraSourceSupplier, failureCallback));
> > +                loadCamera(dialog, cameraSourceSupplier, failureCallback);
>
> Did you actually test that this works?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WPIRoboticsProjects/GRIP/pull/213/files#r47576399>.
>